libs/template/experimental/virtual-scrolling/src/lib/virtual-list-template-manager.ts

```diff
@@ -11,7 +11,8 @@ export interface RxVirtualListChange<T, C> {
 
 export type RxVirtualListChanges<T, C> = [
   number,
-  () => RxVirtualListChange<T, C>
+  () => RxVirtualListChange<T, C>,
+  boolean? // removed
 ][];
```

Comment:
What about using named tuples to be explicit? I'm guessing about the purpose of the elements and the corresponding name but something like:
```ts
export type RxVirtualListChanges<T, C> = [
  itemCount: number,
  onChange: () => RxVirtualListChange<T, C>,
  removed: boolean?
][];
```

Reply:
didn't even know this is a thing